src/textToArt.ts

```diff
@@ -1,17 +1,19 @@
 import AinftBase from './ainftBase';
 import {
   HttpMethod,
+  Task,
   TextToArtResponse,
   TextToArtParams,
   TextToArtTxHash,
+  DiscordMessageInfo,
 } from './types';
 
 export default class TextToArt extends AinftBase {
   /**
-   * Get result of a text to art task.
-   * @param {appId} appId where uses text to art
-   * @param {taskId} taskId
-   * @returns {Promise<TextToArtResponse | null>} Return text to art results or null.
+   * Get the result of a text to art task.
+   * @param {string} appId - The ID of the app where the text to art task was used.
```

Comment:
minor) Since 'text to art' is a service name, I think 'TextToArt' or 'text-to-art' would be better.

Reply:
Yes, I will change it